### PR TITLE
Py3 compatibility

### PR DESCRIPTION
**Pull request by murielll**

### CODE REVIEW

File: megaraid.py [hunk 1]
```diff
@@ -32,11 +38,11 @@ def check_adapter(self, instance, megacli):
                 current_adapter = line.split(' ')[1]
                 adapters['0'] = dict()
             if line.startswith('State'):
-                if "Optimal" in line:
+                if 'Optimal' in line:
                     adapters[current_adapter]['state'] = 1
                 else:
-                    adapters[current_adapter]['state'] = 0
-                    self.send_megaraid_alert('Adapter %s state is not optimal: %s' % (current_adapter, adapters[current_adapter]['state']), current_adapter)
+                    adapters[curent_adapter]['state'] = 0
```
Comment (leucos) [inline megaraid.py]: I think there is  a typo here

**Comment by leucos** [comment]
Thanks a lot @murielll 
I think there is a small typo, but the rest seems great.

File: megaraid.py (hunk 1 above)
Comment (murielll) [reply megaraid.py]: Did you mean double quotes in "Optimal" ?

Comment (leucos) [reply megaraid.py]: No. `curent` instead of `current`.

Comment (murielll) [reply megaraid.py]: ah, sorry, didn't notice that

**Comment by leucos** [comment]
Thanks @murielll

**Comment by murielll** [comment]
> Thanks @murielll

you are welcome!

**Pull request merged**